assets/src/edit-story/components/form/toggle.js

```diff
@@ -23,9 +23,10 @@ import PropTypes from 'prop-types';
 const CheckBoxInput = styled.input.attrs({ type: 'checkbox' })`
   position: absolute;
   opacity: 0;
-  height: 0;
-  width: 0;
-  margin: 0;
+  height: 0 !important;
```

Comment:
What's overriding these?

Reply:
Overriding original wp styling.

Reply:
We should really find a better way for this. See #349. We can use that issue to track this.